bdai_ros2_wrappers/bdai_ros2_wrappers/feeds.py

```diff
@@ -190,6 +197,7 @@ def __init__(
         self,
         feed: MessageFeed,
         fn: Callable[..., MessageT],
+        autostart: bool = True,
```

Comment:
```suggestion
        *,
        autostart: bool = True,
```

Make the use of the code more readable

```
AdaptMessageFeed(..., autostart=False) vs AdaptMessageFeed(..., False)
```